### DISCUSSION TITLE
xDrip auto launches even though setting is off

### DISCUSSION THREAD

**Discussion by jcarrollpe** [NightscoutFoundation/xDrip#3340] [Q&A] [answered]
Hello, I've been using xDrip for a few weeks now. My phone is a rooted Pixel 7 Pro.  Currently I am using it in hardware data source: "Libre (patched app)" with Juggluco as master. Juggluco is configured with my Libreview credentials. Everything has been working without issue but I have noticed that xDrip will always "auto launch" on boot. I disabled this in Settings but it persists. Not only that but if I force close xDrip in settings, the app will respawn itself within one minute. FC again, it will start again.  I would still like to use xDrip off and on while actively testing, but when I switch any app off -and- disable the in-app setting so it should not start, I do not think I should. I was using  the xDrip widget, and thought maybe the widget was calling the app to restart, but the behavior persists even after removing the widget. Now I am out of ammo on how to stop this. Surely this is not intentional, and I may very well be missing a setting somewhere else but I cannot find anything related. Please advise how i can set xDrip so that it will not auto launch on boot or respawn itself when closed manually. Thanks!

**Comment by Navid200** [comment, marked as answer]
It is intentional.
I have diabetes.  I depend on the alerts I get from xDrip if my glucose is too low.  My life depends on it.
I sure don't want it to stop working if I accidentally shut it down instead of minimizing it.

The feature to make xDrip a follower instead of a collector has been added to allow users do that if they want to.  But, xDrip was created to be a CGM, not a follower.  It still serves many people who use it as their main CGM app.  As such, it needs to act like a medical app rather than a game or entertainment app.

**Comment by jcarrollpe** [comment]
Ok, thanks for your quick response. I didn't read that in the docs and was not expecting this behavior.
